manifests/server/kickstart/runpuppet.pp

```diff
@@ -0,0 +1,86 @@
+# This class manages the runpuppet script, which is a script that can be run
+# to bootstrap provisioned systems, adding them to puppet and running it in a
+# fashion similar so `simp bootstrap`.
+#
+# @param data_dir The location of the web root in which runpuppet wil be placed
+#
+# @param location The absolute location of the runpuppet file to be dropped
+#   Use this setting if you're using this module outside of SIMP and are
+#   managing another web server for kickstarting.
+#
+# @param ntp_servers
+#   An array of ntp servers or hash of server/vaule pairs that should
+#   be used during client kickstarts to slew the local time correctly
+#   prior to PKI key distribution.
+#
+#   Failure to set the system clock will not cause the runpuppet script to fail
```

Comment:
Won't failure to set the system clock possibly cause the runpuppet script to fail?